backend/src/services/review.ts

```diff
@@ -89,6 +91,18 @@ export async function createAccessRequestReviews(model: ModelDoc, accessRequest:
   await Promise.all(createReviews)
 }
 
+export async function removeAccessRequestReviews(accessRequestId: string): Promise<DeleteResult> {
+  const deletions = await Review.deleteMany({ accessRequestId: accessRequestId })
+
+  if (deletions.deletedCount === 0) {
```

Comment:
I don't think the is necessarily an error. For example, the model might have been missing the relevant review roles when the access request was made so new reviews were made for that access request.